scala/scala.bzl

```diff
@@ -614,31 +670,33 @@ def _scala_binary_common(ctx, cjars, rjars, transitive_compile_time_jars, jars2l
       files=depset([ctx.outputs.executable]),
```

Comment:
@johnynek can we add here the actual jars? is there a reason to **only** return the executable?

Reply:
@johnynek ping

Reply:
I'm not really sure what needs to be there actually. Since `scala_binary` creates a default executable, maybe it should only be the executable? Maybe it doesn't matter. Honestly, some of the bazel stuff is not super well documented and I wrote some of this about 2 years ago now and it might have both changed in bazel and also I may have misunderstood it at the time.

I'm afraid I don't have a great answer for you as to what should be in there.

Reply:
This is what the java binary does so I'm guessing it's modeled after that.
We found a workaround so the current status quo is ok